index.html

```diff
@@ -98,8 +98,14 @@
   setInterval(function() {
     displayModifiedFiles();
   }, 3000);
+
+  // monitor changes on branch list and tag list, 1 check per second
+  setInterval(function () {
+    refreshList(false);
+  }, 1000);
```

Comment:
Do we really need to monitor the branch and tag list every second? This seems wasteful.

Of course, I could be naive, but why don't you refresh the list when you open the dropdown?

Reply:
@colinwerner yea, it does seem a little bit overkill and it also might affect performance. I will talk with my group members to discuss your idea. Thank you!

Reply:
Hi @colinwerner , 

After talked with my group members and heard other members' opinions. 
Below is my thoughts on the idea to this monitor:
The only drawbacks of this one check per second monitor that I could think of is that it could be slowing down the performance and using a little bit of resources. Since it is only a monitor, it will not refresh the list every one second, it just checks the list if no change, it will simply return. Base on this behavior, I would like to argue the performance will not be affected too much as I had also tested this on small and bigger project repos, and I didn't see any signs of performance change. 
However, The advantage of this monitor is that when user had the list open and he or she did some changes using CLI or other tools on the local repo, my monitor will detect changes immediately and show users the latest list of branches and tags. I think the pro of this monitor out weights the con.
But that is just my opinion on this part. Please feel free to correct me and share your opinions. 

Thank you.

Reply:
Thanks @SiRumCz, that's fine.